### ISSUE TITLE
Handle error message when Wikidata search returns an item with no description 

### ISSUE THREAD

**Issue by lubianat** [lubianat/wdcuration#2]
Currently it fails like this:
```
  File "/home/lubianat/Documents/wiki_related/wdcuration/wdcuration/wdcuration.py", line 54, in parse_wikidata_result
    "description": first_item["description"],
KeyError: 'description'
```

**Comment by jvfe** [comment]
I've noticed you seem to have fixed this (b8cc97a5) but the [version on PyPI](https://pypi.org/project/wdcuration/) is still 0.0.1, therefore it's this version that pyorcidator is currently using - which means bugs when handling missing descriptions, which I found while trying to run the example on lubianat/pyorcidator#50. 

Could you bump the version in PyPI to be the latest one?

**Comment by lubianat** [comment]
@jvfe I think it is done: https://pypi.org/project/wdcuration/0.1.1/

eu ainda me embanano com o PyPI as vezes

**Comment by jvfe** [comment]
Yep, now it works after running `pip install --upgrade wdcuration`. Thanks!
I think this can be closed then.

Relaxa, é mais simples do que parece
